lib/warden/jwt_auth/strategy.rb

```diff
@@ -8,7 +8,9 @@ module JWTAuth
     # `Authorization` request header
     class Strategy < Warden::Strategies::Base
       def valid?
-        !token.nil?
+        issuer_claim_configured? ? issuer_claim_matches? : token_exists?
+      rescue JWT::DecodeError
+        true
```

Comment:
Why? 🤔

Reply:
To maintain backward compatibility. Before, any value in the auth header had the strategy return `true`. Now that we are parsing the token to determine validity, this must be caught.